thirdeye/thirdeye-frontend/app/pods/components/alert-details/component.js

```diff
@@ -1032,25 +1032,22 @@ export default Component.extend({
     edgesLabel,
     color,
     useValue) {
-    // build set of anomalous values (newer of 2 sets of anomalies)
     if (!_.isEmpty(filteredAnomalies) && timeseries && !_.isEmpty(series.Current)) {
       const valuesCurrent = [];
+      // make sure anomalies ascend by startTime
+      filteredAnomalies.sort((a, b) => a.startTime - b.startTime);
+      let anomalyIndex = 0
+      let currentAnomaly = filteredAnomalies[anomalyIndex];
       // needed because anomalies with startTime before time window are possible
-      let currentAnomaly = filteredAnomalies.find(anomaly => {
-        return anomaly.startTime <= series.Current.timestamps[0];
-      });
-      let inAnomalyRange = currentAnomaly ? true : false;
+      let inAnomalyRange = currentAnomaly.startTime <= series.Current.timestamps[0];
```

Comment:
The picky requirement, I suggest using `isInAnomalyRang` for a boolean value. It's a clear variable name to express the variable type. it will enhance code readability.

Reply:
also, please make sure `series` has Current and timestamps properties.

Reply:
totally makes sense